src/views/Statistic/StatisticGroup.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import { customPropTypes, getUnhandledProps, useKeyOnly } from '../../utils/propUtils'
+import numberToWord from '../../utils/numberToWord'
+import META from '../../utils/Meta'
+import Statistic from './Statistic'
+
+function StatisticGroup(props) {
+  const { children, className, horizontal, items, widths } = props
+  const classes = cx(
+    'ui',
+    useKeyOnly(horizontal, 'horizontal'),
+    _.isNumber(widths) ? numberToWord(widths) : widths,
+    className,
+    'statistics'
+  )
+  const rest = getUnhandledProps(StatisticGroup, props)
+
+  if (!items) {
+    return <div className={classes} {...rest}>{children}</div>
+  }
+
+  let itemsJSX = []
+
+  items.forEach((item, index) => {
+    const key = `${item.label}-${item.value}-${index}`
+    itemsJSX.push(<Statistic key={key} {...item} />)
+  })
+
+  return <div className={classes} {...rest}>{itemsJSX}</div>
```

Comment:
As noted in Statistic, spread before applying className to prevent user's className from overriding the className build up.

Conformance tests ensure props are spread, I should add a test to ensure className is not override  during spread ;)